calicoctl/commands/migrate/unlock.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2020 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package migrate
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/docopt/docopt-go"
+
+	"github.com/projectcalico/calicoctl/calicoctl/commands/clientmgr"
+	"github.com/projectcalico/calicoctl/calicoctl/commands/constants"
+	"github.com/projectcalico/libcalico-go/lib/options"
+)
+
+func Unlock(args []string) error {
+	doc := `Usage:
+  calicoctl migrate unlock [--config=<CONFIG>]
```

Comment:
I think we said this should be `calicoctl datastore migrate lock`?

and `calicoctl datastore migrate import/export`?

Reply:
Oops, glanced over it. I'll fix it.